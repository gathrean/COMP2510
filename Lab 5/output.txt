1
A2
0000011
A023.txt
A00050.txt
A10001