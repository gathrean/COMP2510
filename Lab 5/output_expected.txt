A1001.txt
a3333.txt
A00010001.txt
A33333
34222